packages/ds-component/src/index.ts

```diff
@@ -384,6 +384,34 @@ export const objectTransform: ObjectTransform = (message: Message) => ({
   interactions: transformDSInteraction(message),
 });
 
+/**
+ * Check if the transform is likely the identity function
+ * This is not a supported implementation path
+ * Avoid this if at all possible - please use either objectTransform or tableTransform
+ */
+const isIdentityFunction = (transform): boolean => {
```

Comment:
nit: `isProbablyIdentityFunction` is a slightly better name. Especially when you're just looking at the callsite.

Reply:
done